docs/src/operations/guides/validator-failover.md

```diff
@@ -13,13 +13,126 @@ without any safety issues that would otherwise be associated with running two
 instances of your validator.
 
 You will need two validator-class machines for your primary and secondary
-validator. A third machine for running an [etcd](https://etcd.io/) cluster,
-which is used to store the tower voting record for your validator.
+validator. 
+
+## Manual Identity Transition
+
+[Based on Identity Transition by Pumpkin's Pool](https://pumpkins-pool.gitbook.io/pumpkins-pool)
```

Comment:
While I appreciate the shout out and I'd love to have my validator's name in the official docs, I think it might set a bad precedent. I don't know if we want to start polluting the docs with any specific validator branding. I like how they are currently concise and agnostic.

Honestly, I do appreciate the credit, but maybe we should just focus on the transition process without including this section.

Reply:
That's reasonable. This was still a rough draft and I was going to ask for feedback in general. I only included it cause this was a nearly direct copy and paste of someone else's work. I had permission, but still doesn't feel right to not give credit.

Reply:
Including a note somewhere in this PR though (such as these comments) could literally be the credit though. People can find this came from you, but it is not direct branding.

Reply:
I also honestly forgot that is your validator's name. I see you in the discord under that name so often that I just connect that name with you, not your validator.

Reply:
> I also honestly forgot that is your validator's name. I see you in the discord under that name so often that I just connect that name with you, not your validator.

Haha no worries. I appreciate you giving credit!